cfg/qt.cfg

```diff
@@ -1332,21 +1332,23 @@
   <define name="QT_TRANSLATE_NOOP_UTF8(scope, x)" value="x"/>
   <define name="QT_TRANSLATE_NOOP3(scope, x, comment)" value="{x, comment}"/>
   <define name="QT_TRANSLATE_NOOP3_UTF8(scope, x, comment)" value="{x, comment}"/>
-  <define name="QCOMPARE(a,b)" value=""/>
-  <define name="QVERIFY(expr)" value=""/>
-  <define name="QVERIFY2(cond, msg)" value=""/>
+  <define name="QCOMPARE(actual, expected)" value="(actual) == (expected)"/>
```

Comment:
This will propably produce other warnings. I assume a constStatement-warning could be issued when these changes are applied. A poential fix would be to rewrite the macros as follows:

`value="{(void)actual; (void)expexted;}"`

@danmar and others: are you fine with this suggestion?

Reply:
hmm.. I just made an alternative fix. feel free to review my fix.

Reply:
Yes I have seen some of this warnings, your suggestion fixes this constStatement warning.
for example when comparing:
QCOMPARE(non_const_function(), var)

I also have seen a new ignoredReturnValue-Warning with QVERIFY

QVERIFY(QFile::exists(fullFilePath));
cause an ignoredReturnValue-warning.

Any suggestion how to fix it?

Reply:
Two options:
 * that might be a false positive. If you have an explicit cast to void it seems the warning should not be written. Or at least we can have a separate warning for that.
 * rewrite the macro. Maybe it's enough to add some `!= 0` or something in the macro?

Reply:
The false positive are gone with your alternative fix:

<define name="QVERIFY(expr)" value="(void)(expr)"/>